shopelectro/tests/tests_views.py

```diff
@@ -522,3 +525,29 @@ def test_admin_autocomplete_has_no_model_pages(self):
             f'?term={self.QUOTED_SIGNLE_RESULT_TERM}&pageType=category'
         )
         self.assertTrue(len(json_to_dict(response)) == 1)
+
+
+class Redirects(TestCase):
+
+    fixtures = ['dump.json']
+
+    # will be resurrected at rf#140
+    @unittest.expectedFailure
+    def test_redirect_on_existing_page(self):
+        """DB based redirect from existing url should do, but should not avoid it."""
```

Comment:
"""DB based redirect from existing url should do, but should not avoid it."""
-> 
"""Redirect middleware of refarm-site redirects from existing url too"""

decide by yourself

Reply:
`Refarm-site's custom redirects app should redirect from existing url too`

TODO